diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index edc31461d..49ac5011d 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1151,7 +1151,7 @@ target_compile_definitions(${VERSION_PROJECT} PRIVATE
     CHATTERINO_GIT_COMMIT=\"${GIT_COMMIT}\"
     CHATTERINO_GIT_MODIFIED=${GIT_MODIFIED}
 
-    CHATTERINO_CMAKE_GEN_DATE=\"${cmake_gen_date}\"
+    CHATTERINO_CMAKE_GEN_DATE=\"$ENV{GEN_DATE}\"
 )
 
 target_link_libraries(${LIBRARY_PROJECT} PRIVATE ${VERSION_PROJECT})
